gitbutler-ui/src/lib/components/CommitDialog.svelte

```diff
@@ -134,10 +134,21 @@
 					on:input={useAutoHeight}
 					on:focus={useAutoHeight}
 					on:change={() => currentCommitMessage.set(commitMessage)}
+					on:keydown={(e) => {
+						if ((e.ctrlKey || e.metaKey) && e.key === 'Enter') {
+							if ($expanded) {
+								if (commitMessage) {
```

Comment:
Let's move this check into the commit function? That way we won't need an intermediary function to make things dry.

Reply:
The `commit` function is called on button click, line 215. 

Do you want me to modify the `commit` function to handle both the click and keyboard event. I could do this by passing a parameter but it won't look pretty.

Reply:
What you have now looks good to me? I'm not sure what parameter you would want to pass.